data/ames_mutagenicity/meta.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: ames_mutagenicity
+description: |-
+    Mutagenicity means the ability of a drug to induce genetic alterations.
+    Drugs that can cause damage to the DNA can result in cell death or other severe
+    adverse effects. Nowadays, the most widely used assay for testing the mutagenicity
+    of compounds is the Ames experiment which was invented by a professor named
+    Ames. The Ames test is a short term bacterial reverse mutation assay detecting
+    a large number of compounds which can induce genetic damage and frameshift mutations.
+    The dataset is aggregated from four papers.
+targets:
+    - id: mutagenic
+      description: whether it is mutagenic (1) or not mutagenic (0)
+      units:
+      type: boolean
+      names:
+          - mutagenicity
```

Comment:
@MicPie 

I'm not sure if this is a good way of doing it or if we can be more flexible, but I tried to write the names in a form that we can use them as shown below. But I'm very open to changes 
<img width="952" alt="Screenshot 2023-04-20 at 22 49 48" src="https://user-images.githubusercontent.com/32935233/233484747-6614b3e0-a875-4c80-ac1e-f0597fd0934e.png">

Reply:
otherwise, looks very good to me